pom.xml

```diff
@@ -19,94 +19,280 @@
-      </dependency>
-      <!-- Video Calls extension -->
-      <dependency>
-        <groupId>org.exoplatform.addons.gamification</groupId>
-        <artifactId>gamification-services</artifactId>
-        <version>${project.version}</version>
-        <type>jar</type>
-      </dependency>
-      <dependency>
-        <groupId>org.exoplatform.addons.gamification</groupId>
-        <artifactId>gamification-portlets</artifactId>
-        <version>${project.version}</version>
-        <type>war</type>
-      </dependency>
-      <dependency>
-        <groupId>org.exoplatform.addons.gamification</groupId>
-        <artifactId>gamification-extension</artifactId>
-        <version>${project.version}</version>
-        <type>war</type>
-      </dependency>
-      <!-- for tests -->
-      <dependency>
-        <groupId>junit</groupId>
-        <artifactId>junit</artifactId>
-        <version>${junit.version}</version>
-        <scope>test</scope>
-      </dependency>
-    </dependencies>
-  </dependencyManagement>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>addons-parent-pom</artifactId>
+        <groupId>org.exoplatform.addons</groupId>
+        <version>8-RC01</version>
+    </parent>
+
+    <groupId>org.exoplatform.addons.gamification</groupId>
+    <artifactId>gamification</artifactId>
+    <version>1.0.x-SNAPSHOT</version>
+    <packaging>pom</packaging>
+    <name>eXo Gamification</name>
+    <description>eXo Gamification extension</description>
+
+    <modules>
+        <module>services</module>
+        <module>portlets</module>
+        <module>extension</module>
+        <module>packaging</module>
+    </modules>
+
+    <scm>
+        <connection>scm:git:git://github.com/exo-addons/gamification.git</connection>
+        <developerConnection>scm:git:git@github.com:exo-addons/gamification.git</developerConnection>
+        <tag>HEAD</tag>
+        <url>https://github.com/exo-addons/gamification</url>
+    </scm>
+
+    <properties>
+        <!-- Defaut compiler settings -->
+        <maven.compiler.optimize>true</maven.compiler.optimize>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+        <!-- eXo Modules -->
+        <!-- **************************************** -->
+        <!-- Dependencies versions                    -->
+        <!-- **************************************** -->
+        <org.exoplatform.depmgt.version>13-RC01</org.exoplatform.depmgt.version>
+        <org.exoplatform.platform.version>5.0.0</org.exoplatform.platform.version>
+        <!-- for tests -->
+        <junit.version>4.12</junit.version>
+        <!-- Liquibase stack -->
+        <org.liquibase.version>3.5.5</org.liquibase.version>
```

Comment:
Can't you use the same version than PLF ?

Reply:
I really don't remember why I have used this version (I'll keep this version and in V2 I'll dig in to figure out why I used it)